WORKSPACE

```diff
@@ -35,13 +35,11 @@ http_archive(
 
 # rules_nodejs
 # To use the JavaScript version of Sass, we need to first install nodejs
-rules_nodejs_version = "1.4.1"
+rules_nodejs_version = "4.6.1"
```

Comment:
Is 4.6.1 the latest version? 
How risky is this upgrade?

Reply:
4.6.1 is the latest version that doesn't introduce breaking changes with the way that rules_nodejs is being used in this project. (latest available version is 6.0.1)